src/mame/layout/tek4404.lay

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+authors: Elektraglide
+
+Tektronix 4404 Artificial Intelligence System.
+-->
+<mamelayout version="2">
+	<element name="static_black"><rect><color red="0.0" green="0.0" blue="0.0" /></rect></element>
+	<element name="led_round" defstate="1">
+		<disk state="1">
+			<color red="0.2" green="0.0" blue="0.0" />
+		</disk>
+		<disk state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+		</disk>
+	</element>
+	<element name="led_rect" defstate="1">
+		<rect state="1">
+			<color red="0.2" green="0.0" blue="0.0" />
+		</rect>
+		<rect state="0">
+			<color red="1.0" green="0.1" blue="0.1" />
+		</rect>
+	</element>
+	<view name="Graphics">
+		<bounds x="0" y="0" width="640" height="496" />
+		
+		<screen index="0">
+			<bounds x="0" y="0" width="640" height="480" />
+		</screen>
+		<element name="spacer" ref="static_black">
+			<bounds x="0" y="480" width="640" height="16" />
+		</element>
```

Comment:
You don’t really need this thing as you’ve got explicit bounds for the view, and the LEDs will draw on top of the black background anyway.  But if you insist on using it, you’re better off putting it behind the screen rather than on top to ensure it doesn’t slightly crop the bottom of the screen when the floating point view coordinates are quantised to whole pixel values.